cropduster/tests/helpers.py

```diff
@@ -39,26 +39,6 @@ def setUp(self):
         self.TEST_IMG_DIR = ORIG_IMG_PATH
         self.TEST_IMG_DIR_RELATIVE = os.path.join(random, 'data')
 
-    def assertImageColorEqual(self, element, image):
```

Comment:
Why was this deleted?

Reply:
To explain why it's here: this was written to catch a common class of bug, which is that after uploading and cropping, the image preview might not appear.

Reply:
I just re-enabled this check and ran it locally, and I think this caught a real bug in this PR. In addition to this failing with `AssertionError: Colors differ: #ffffff != #000000` I see in the console the error `Failed to load resource: the server responded with a status of 404 (Not Found) _preview.png`

Reply:
Sorry to be carrying on a conversation with myself, but I was curious about this and looked into it more: this particular bug is not a real failure, though I'm curious whether  It seems that the issue here is that `_preview.ext` doesn't get created by `generate_thumbs`. The quickest way to fix would be to add some code to `create_unique_image` in the cropduster test helpers that also saves a file to that path (it doesn't really matter what the dimensions are, it can be the same as the original for the purpose of these checks).

Reply:
It didn't cause an issue before because it would use the first available thumb, instead of `_preview.ext`. When the change was made in this PR to cropduster.js to use _preview.ext instead, that introduced the test failure.

Reply:
Reinstated the test and added shtuff to make a preview image in `create_unique_thumbs` as suggested. Got rid of the test initially because I knew the failure was spurious (based on manual upload tests) and..I was too lazy to dig much further :sweat_smile: